Gemfile

```diff
@@ -5,9 +5,8 @@ gemspec
 gem 'rake'
 
 group :development do
-  gem 'byebug', platform: :ruby
   gem 'danger-changelog', '~> 0.2.1'
-  gem 'rubocop', '0.56.0'
+  gem 'rubocop'
```

Comment:
Rubocop upgrades should be a separate pull request as it often involves more discussions around preferable syntax/pattern in the project.